subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -1,837 +1,346 @@
-These will be _ignored_ by Gradle, which will only use the repositories declared in the build itself.
+Gradle provides built-in shorthand notations for certain widely-used repositories.
+Popular repositories include <<#sub:maven_central,Maven Central>> and <<#sub:maven_google,Google's Maven repository for Android>>.
+For a full list of built-in shorthand notations, see link:{groovyDslPath}/org.gradle.api.artifacts.dsl.RepositoryHandler.html[RepositoryHandler].
+To declare a repository that doesn't have a built-in shorthand notation, declare the repository by URL.
+
+[[sub:maven_central]]
+=== Declare the Maven Central Repository
+
+The Maven Central repository hosts open source libraries for Java projects.
+To add the link:https://repo.maven.apache.org/maven2/[Maven Central repository] to your build, add the following to your build script:
 
-[NOTE]
 ====
-This is a reproducibility safe-guard but also a security protection.
-Without it, an updated version of a dependency could pull artifacts from anywhere into your build.
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-central]"]
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-central]"]
 ====
 
-[[sec:repository-types]]
-== Supported repository types
+[[sub:maven_google]]
+=== Declare the Google Maven Repository
 
-Gradle supports a wide range of sources for dependencies, both in terms of format and in terms of connectivity.
-You may resolve dependencies from:
+The Google Maven repository hosts Android-specific artifacts, including the Android SDK.
+To add the link:https://maven.google.com/[Google Maven repository] to your build, add the following to your build script:
 
-* Different formats
-** a <<#sec:maven_repo,Maven compatible>> artifact repository (e.g: Maven Central)
-** an <<#sec:ivy_repositories,Ivy compatible>> artifact repository (including custom layouts)
-** <<#sub:flat_dir_resolver,local (flat) directories>>
-* with different connectivity
-** <<#sec:authentication_schemes, authenticated repositories>>
-** a wide variety of <<#sec:supported_transport_protocols,remote protocols>> such as HTTPS, SFTP, AWS S3 and Google Cloud Storage
+====
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-google]"]
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-google]"]
+====
 
 [[sub:flat_dir_resolver]]
-=== Flat directory repository
+== Declare a Flat Directory Repository
 
-Some projects might prefer to store dependencies on a shared drive or as part of the project source code instead of a binary repository product. If you want to use a (flat) filesystem directory as a repository, simply type:
+Some projects store dependencies on a shared drive or as part of the project source code instead of a binary repository product.
+Gradle calls this a directory repository.
```

Comment:
I will leave this one for the technical review. The original document sometimes omits "flat" or uses it in a parenthetical. I eliminated it here because "directory repository" feels like enough of a description and generally fewer words == better, more readable doc. However, the API does use the term `flatDir`, so maybe we should emphasize "flat" more.